src/events/mongodb.js

```diff
@@ -24,21 +24,49 @@ function stringifyData(obj) {
     return data.substring(0, MAX_DATA_LENGTH);
 }
 
+/**
+ * Extracts connection details from a conenction
+ * @param {Object} connId The connectionId object of the instrumented event
+ * @return {{port: *, host: *}} extracted connection details
+ */
+function getConnectionDetails(connId) {
+    let host;
+    let port;
+    utils.debugLog('Epsagon - MongoDB - inspecting connectionId', connId);
+    if (connId) {
+        if (typeof connId === 'string') {
+            const parts = connId.split(':');
+            if (parts.length && parts[0][0] === '/') {
+                host = 'localhost';
```

Comment:
Can you add a quick explaination what is this scenario? (What's the original strong looks like)

Reply:
in case it is a unix domain socket (i.e a file on the FS)